docs/rules/prefer-node-protocol.md

```diff
@@ -11,6 +11,12 @@ When importing builtin modules, it's better to use the [`node:` protocol](https:
 
 And don't forget to [upvote this issue](https://github.com/nodejs/node/issues/38343) if you agree.
 
+Note that this feature is not available in early versions of Node 12 and Node 14:
+
+> v16.0.0, v14.18.0 (ESM import and CommonJS require())
```

Comment:
Why not mention Node 18 here as well?

Reply:
This text was copied directly from https://nodejs.org/api/esm.html#node-imports and is only supposed to mention the initial Node versions that supported this feature. Since Node 16 has full support for it, any subsequent versions can be assumed to fully support it as well.